### ISSUE TITLE
Semaphore not recognizing installed Ansible

### ISSUE THREAD

**Issue by jrglynn2** [ansible-semaphore/semaphore#1252]
I just installed semaphore, and am getting the following error:
```
1:44:01 PM
File "/home/ec2-user/.local/bin/ansible-playbook", line 5, in
1:44:01 PM
from ansible.cli.playbook import main
1:44:01 PM
ModuleNotFoundError: No module named 'ansible'
1:44:01 PM
Running playbook failed: exit status 1
```

I do not use `venv` for either semaphore or ansible. Running ansible commands from the command line works fine, but it seems like Semaphore doesn't see ansible for some reason. You can see that it finds `ansible-playbook`, but `ansible-playbook` then can't seem to find `ansible`.

Here is my systemd service file. I tried adding the `Environment` line, which is a copy of my normal path, and `ec2-user` is the user we use to run ansible commands from server command line.
```
[Unit]
Description=Semaphore Ansible
Documentation=https://github.com/ansible-semaphore/semaphore
Wants=network-online.target
After=network-online.target

[Service]
Type=simple
Environment="PATH=/home/ec2-user/.local/bin:/home/ec2-user/bin:/usr/local/bin:/usr/bin:/usr/local/sbin:/usr/sbin"
User=ec2-user
Group=ec2-user
ExecReload=/bin/kill -HUP $MAINPID
ExecStart=/usr/bin/semaphore service --config=/opt/semaphore/config/config.json
SyslogIdentifier=semaphore
Restart=always

[Install]
WantedBy=multi-user.target
```

_Originally posted by @jrglynn2 in https://github.com/ansible-semaphore/semaphore/issues/1075#issuecomment-1554834360_

**Comment by jrglynn2** [comment]
System Information:

OS:  `Red Hat Enterprise Linux release 8.7 (Ootpa)`

Semaphore Version:  `v2.8.90`

Ansible Version:
```
Ansible Full Version:  7.3.0
Ansible-Core Version:  ansible [core 2.14.4]
  config file = /opt/ansible/ansible.cfg
  configured module search path = ['/home/ec2-user/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /home/ec2-user/.local/lib/python3.9/site-packages/ansible
  ansible collection location = /opt/ansible/ansible_collections
  executable location = /home/ec2-user/.local/bin/ansible
  python version = 3.9.13 (main, Nov  9 2022, 13:16:24) [GCC 8.5.0 20210514 (Red Hat 8.5.0-15)] (/usr/bin/python3)
  jinja version = 3.0.2
  libyaml = True
```

**Comment by jrglynn2** [comment]
I found a solution for this issue.

Ansible for me was installed as a user (`executable location = /home/ec2-user/.local/bin/ansible`) . For some reason using `ec2-user` as User & Group in the service file wasn't enough, so I had to add a PYTHONPATH variable (https://docs.python.org/3/using/cmdline.html#envvar-PYTHONPATH).

In order to get the proper path to use for PYTHONPATH, I ran `python -c "import site; print(site.USER_SITE)"` when logged in as `ec2-user`.

So, I ended up adding this line to my service file:
`Environment=PYTHONPATH=/home/ec2-user/.local/lib/python3.9/site-packages`

Closing issue.
